node-express/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:9-alpine
+ENV NODE_PATH=/usr/local/lib/node_modules
+
+RUN mkdir -p /var/server
+
+WORKDIR /var/server
+
+RUN npm install -g express@4.16.3
```

Comment:
Perhaps include a default `CMD ["node", "/app/server.js"]` or something that you could standardize on? The java base image expects `/app/app.jar` by default, for example.

Reply:
God idè!